paimon-spark/paimon-spark-3.1/src/test/java/org/apache/paimon/spark/SparkGenericCatalogTest.java

```diff
@@ -105,6 +107,42 @@ public void testCsvTable() {
                 .containsExactlyInAnyOrder("[1,2,3]", "[4,5,6]");
     }
 
+    @Test
```

Comment:
And I prefer to add these UTs in `DDLTestBase`.

Reply:
already moved to DDLTestBase

Reply:
already moved to DDLTestBase